Functions/xASL_test_GetLogContent.m

```diff
@@ -0,0 +1,308 @@
+function [logContent] = xASL_test_GetLogContent(rootDir, printContent, storeRelativePath, exportTable)
+%xASL_test_GetLogContent Get warnings and errors from log files
+%
+% FORMAT: [logContent] = xASL_test_GetLogContent(rootDir, [printContent], [storeRelativePath], [exportTable])
```

Comment:
In other files, we wrote - not sure how relevant
[printContent, storeRelativePath, exportTable]
instead of 
[printContent], [storeRelativePath], [exportTable]